src/renderer/components/ListItem.vue

```diff
@@ -1,7 +1,21 @@
 <template>
-  <div class="relative block bg-gray-50 rounded-md p-3 mb-2 hover:bg-indigo-50">
-    <p class="uppercase tracking-wide font-semibold text-xs text-gray-600">Recorded at</p>
-    <p class="uppercase tracking-wide font-bold text-md text-gray-800">12:34 PM</p>
+  <div class="flex flex-row relative block bg-gray-50 rounded-md p-3 mb-2 hover:bg-indigo-50">
+    <div class="flex flex-1 items-center">
+      <div class="flex-1">
+        <p class=" uppercase tracking-wide font-semibold text-xs text-gray-600">Recorded at</p>
+        <p class=" uppercase tracking-wide font-bold text-md text-gray-800">12:34 PM</p>
+      </div>
+      <button class="flex flex-col rounded-md w-10 h-10 text-indigo-600 hover:stroke-current hover:text-white hover:bg-indigo-500 hover:border-transparent focus:outline-none active:bg-indigo-700 justify-center items-center mr-3">
```

Comment:
Might I suggest that we turn the button into an `IconButton` component? This would keep the `ListItem` component cleaner and possibly allow for reuse to implement the preferences button.

Reply:
> Might I suggest that we turn the button into an `IconButton` component? This would keep the `ListItem` component cleaner and possibly allow for reuse to implement the preferences button.

sounds good, i'll add the changes in a bit

Reply:
@chuabingquan currently i have this issue where since the icons used are SVGs, if i were to save it as .svg files and import it using `<img>` instead of using inline SVG, it can't be styled using css (such as for changing the colour on `:focus` and `:hover`)

i'll create a new issue to come back to this in the future; feel free to suggest any solutions or alternatives to this

Reply:
Sure thing, sounds good otherwise!